modules/tiled-gallery/tiled-gallery.php

```diff
@@ -17,6 +17,7 @@ public function __construct() {
 		add_filter( 'jetpack_gallery_types', array( $this, 'jetpack_gallery_types' ), 9 );
 		add_filter( 'jetpack_default_gallery_type', array( $this, 'jetpack_default_gallery_type' ) );
 
+		jetpack_register_block( 'tiled-gallery' );
```

Comment:
I'm a bit unsure if I should instead do this:

```php
if ( $this->tiles_enabled() ) {
	jetpack_register_block( 'tiled-gallery' );
} else {
	jetpack_register_block( 'tiled-gallery', array(), array( 'available' => false, 'unavailable_reason' => 'missing_module' ) );
}
```

Reply:
The idea behind `missing_module` is that we would have a component that would just allow us to show a toggle to the user to activate the missing module. I am not sure this would work the same way.

I am not too familiar with the tile gallery code and how that would work. 
I think the check makes sense but I would just be careful to make sure that reason still make sense in this context.

Reply:
Thanks for elaborating @enejb 

>I am not too familiar with the tile gallery code and how that would work.

Me neither and it seems like we could leave this for future iterations.